src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -455,6 +461,9 @@ class CollectedHeap : public CHeapObj<mtGC> {
   // Default implementation does nothing.
   virtual void print_tracing_info() const = 0;
 
+  double elapsed_gc_cpu_time() const;
+  void log_gc_cpu_time() const;
```

Comment:
They can be private, I believe.

Reply:
`elapsed_gc_cpu_time()` will be callable through an MXBean in a future patch I'm planning, but can make `log_gc_cpu_time()` private